squire_lib/src/tournament.rs

```diff
@@ -1269,6 +1273,62 @@ mod tests {
             .assume_pair();
     }
 
+    #[test]
+    fn create_round_test() {
+        let mut tourn =
+            Tournament::from_preset("Test".into(), TournamentPreset::Swiss, "Test".into());
+        assert_eq!(tourn.pairing_sys.common.match_size, 2);
+
+        fn make_player(tourn: &mut Tournament) -> PlayerId {
+            tourn
+                .apply_op(Utc::now(), TournOp::RegisterPlayer(spoof_account()))
+                .unwrap()
+                .assume_register_player()
+        }
+        let default_salt = DateTime::<Utc>::MIN_UTC;
+
+        let players_too_large = std::iter::repeat_with(|| make_player(&mut tourn))
+            .take(4)
+            .collect_vec();
+        let players_too_small = vec![make_player(&mut tourn)];
+        let players_none = vec![];
+        let players_correct = std::iter::repeat_with(|| make_player(&mut tourn))
+            .take(2)
+            .collect_vec();
+        let repeated_players = std::iter::repeat(make_player(&mut tourn))
+            .take(2)
+            .collect_vec();
+
+        assert!(tourn.start().is_ok());
+
+        assert_eq!(
+            tourn.create_round(default_salt, players_too_large),
```

Comment:
Nit: Its generally easier to use `Utc::now()`:
```suggestion
            tourn.create_round(Utc::now(), players_too_large),
```